core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -788,6 +791,9 @@ object KafkaConfig {
   val ConsumerGroupMaxSizeDoc = "The maximum number of consumers that a single consumer group can accommodate."
   val ConsumerGroupAssignorsDoc = "The server side assignors as a list of full class names. The first one in the list is considered as the default assignor to be used in the case where the consumer does not specify an assignor."
 
+  /** Share Group Configs */
```

Comment:
Any reason of not defining the placeholder as above i.e. `Configurations **/`, if not then:
```suggestion
  /** Share Group Configurations * */
```

Reply:
ack